content/800-data-platform/300-data-proxy/900-considerations-limitations.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Considerations and limitations'
+metaTitle: 'Considerations'
+metaDescription: 'When you use Data Proxy for applications in serverless and edge runtimes, you should bear in mind a few considerations and learn about the current limitations.'
+tocDepth: 2
+---
+
+## Considerations
+
+### Connection to databases hosted with a provider different from AWS
+
+It is possible to connect to a database hosted on any provider, but it needs to be accessible from the public internet. For best results and response times, you should configure the proxy in a region that is the same or as close as possible to the database.
```

Comment:
>it needs to be accessible from the public internet.

I think there is potential for confusion around this. The database needs to be accessible from public IP addresses but it can be technically just our egress IP addresses. I think it could be worth making it more specific, e.g. 

>but it needs to be accessible from the public internet

Suggestion:

>It is possible to connect to a database hosted on any provider, as long as it is accessible using a public internet address.